templates/kubernetes/docs/1.20/charm-containerd.md

```diff
@@ -84,7 +84,7 @@ Registry credentials. Setting this config allows Kubelet to pull images from
 registries where auth is required.
 
 The value for this config must be a JSON array of credential objects, like this:
-`[{"url": "https://my.registry:port", "username": "user", "password": "pass"}]`
+  ``[{"url": "https://my.registry:port", "username": "user", "password": "pass"}]``
```

Comment:
Are you sure you want double quotes here?

Reply:
Ah, well spotted. I was adding them to multiple places and I guess I did this one twice. fixed